### PR TITLE
Adding Melissa to Ways of Working

### PR DESCRIPTION
**Pull request by BatoolMM**
### Summary

This PR is to add Melissa to Ways of Working document

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by BatoolMM** [comment]
Hi @acocac, I wanted to update the Ways of Working document, would you mind reviewing this PR when you have time later on, pls?

File: ways_of_working.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ It further outlines expectations and responsibilities with regard to working on
 |  Batool Almarzouq <br>([BatoolMM](https://github.com/BatoolMM)) | Postdoctoral Researcher, KAIMRC and Honorary Research Fellow, The University of Liverpool  | The Turing Way Translation and Localisation Co-Lead | 2020 - Present | - |
 | Alejandro Coca Castro <br>([acocac](https://github.com/acocac)) | Postdoctoral Research Associate, The Alan Turing Institute | The Turing Way Translation and Localisation Co-Lead | 2021 - Present | - |
```
Comment (acocac) [inline ways_of_working.md:34]:
```suggestion
| Alejandro Coca Castro <br>([acocac](https://github.com/acocac)) | Research Fellow, The Alan Turing Institute | in-kind contribution, Environmental Data Science book Co-Lead | 2021 - Present | The Turing Way Translation and Localisation Co-Lead, 2021 - May 2023 (in-kind) |
```

File: ways_of_working.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ It further outlines expectations and responsibilities with regard to working on
...
 | Andrea Sánchez-Tapia <br>([AndreaSanchezTapia](https://github.com/AndreaSanchezTapia)) | Independent researcher | The Turing Way Translation and Localisation Co-Lead | 2021 - Present | - |
+| Melissa Black <br>([melibleq](https://github.com/melibleq)) | Infrastructure and Impact Measurement Manager at Metadocencia | The Turing Way Translation and Localisation Co-Lead | 2023 - Present | - |
```
Comment (acocac) [inline ways_of_working.md:36]: Welcome Melissa (:

**Review by acocac — APPROVED** [review]
@BatoolMM thanks for adding Melissa, I've updated my details too.

**Review by aleesteele — APPROVED** [review]
Looks great, thanks @BatoolMM, @acocac, & @melibleq !

**Comment by BatoolMM** [comment]
Thank you @acocac and @aleesteele 🚀

File: ways_of_working.md (hunk 2 above)
Comment (melibleq) [reply ways_of_working.md:36]: Thank you so much, @BatoolMM, @acocac and @aleesteele! :)!

**Pull request merged**